changelog.dd

```diff
@@ -45,6 +46,28 @@ $(LI $(LNAME2 asm-attributes, $(D asm) statements can now be used in $(D pure),
 
 $(BUGSTITLE Library Changes,
 
+$(LI $(LNAME2 faster-gc-1, $(BIG The garbage collector is much faster.))
```

Comment:
The performance gain still depends on the application, not all benchmarks show a notable improvement. So I'd be a little more humble and remove the "BIG" (being the first item makes this change sufficiently prominent) and rather make the second line the title, e.g. "GC optimizations".